src/lightning_app/utilities/cli_helpers.py

```diff
@@ -119,3 +121,15 @@ def _retrieve_application_url_and_available_commands(app_id_or_name_or_url: Opti
                     raise Exception(f"The server didn't process the request properly. Found {resp.json()}")
                 return lightningapp.status.url, _extract_command_from_openapi(resp.json())
     return None, None
+
+
+def _arrow_time_callback(_ctx, _param, value, arw_now=arrow.utcnow()) -> arrow.Arrow:
+    try:
+        return arw_now.dehumanize(value)
+    except:
+        pass
```

Comment:
Shall here be some warning at least...

Reply:
We have a common Exception raising below, but sure, having it here will be better!